x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -141,8 +146,7 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
       }
       return null;
     },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [browserFields, globalFilters, globalQuery, indexPatterns, kibana, to, from]
```

Comment:
This was the culprit for the Building Block Rules not working with bulk action issue. We were missing `defaultFilters`, so they were never getting updated within this scope when the user would select `Show building block rules` within the UI.